### DISCUSSION TITLE
please add chants for pony diffusion

### DISCUSSION THREAD

**Discussion by Ineman** [DominikDoom/a1111-sd-webui-tagcomplete#294] [Ideas]
{
        "name": "Pony-Positive",
        "terms": "Pony,Score,Positive,Quality",
        "content": "score_9, score_8_up, score_7_up, score_6_up, source_anime, source_furry, source_pony, source_cartoon",
        "color": 1
    },
{
        "name": "Pony-Negative",
        "terms": "Pony,Score,Negative,Quality",
        "content": "score_1, score_2, score_3, score_4, score_5, source_anime, source_furry, source_pony, source_cartoon",
        "color": 3
    },

I added all source because writing is annoying but erasing is simple.

**Comment by DominikDoom** [comment]
Added in commit e9a21e7.

Just as a side note: I don't have anything against this usage, but personally I feel the webui's built-in style feature is more suited for this.
For example, I have a style set up like this:
![image](https://github.com/DominikDoom/a1111-sd-webui-tagcomplete/assets/34448969/992c6b1a-3e7c-4d3e-806b-fe253e7aa19d)
And selecting that in the dropdown will apply both negative and positive prompt without cluttering the prompt input itself.
With the {prompt} placeholder it even has the correct order so that Pony's positive terms come first and are thus weighted higher, while it's the other way around for negative.

**Reply by Ineman** [reply]
/Thank you for update. I am using gpu cloud such as runpod or vast.ai, not my own local computer. So it's difficult to set styles to newly installed webui environment.
